Integrations/integration-McAfee-ATD.yml

```diff
@@ -36,687 +37,665 @@ configuration:
-        var summary = jres.Summary;
-        summary.VerdictDescription = summary.Verdict.Description;
-        summary.VerdictSeverity = summary.Verdict.Severity;
+        else:
+            demisto.results(res)
 
-        var ec = buildReportContext(summary, uploadData, status, threshold);
-        var jresString = JSON.stringify(jres);
+    def logout():
+        res = sendRequest('/php/session.php','delete', getHeaders())
 
-        if (type === 'json'){
-            summary = formatTableValues(summary);
-            var md = resToMD(summary,'ATD Sandbox Report');
-            return{
-                content: jresString,
-                md: md,
-                ec: ec
-            };
-        }
+    ''' EXECUTION '''
+    LOG('command is %s' % (demisto.command(), ))
 
-        var res = sendRequestRaw('/php/showreport.php?'+uriSuffix+'&iType='+type, 'GET', getHeaders());
-        if (type === 'pdf' || type === 'zip') {
-            if(type === 'pdf' && !getSessionCredentials().isAdmin){
-                return 'No permission to download PDF';
-            }
-            else{
-                filename = filename + '.' + type;
-                return{
-                    content: res.Bytes,
-                    filename: filename,
-                    ec: ec
-                };
-            }
-        }
-        if (type === 'sample') {
-            if (res.Body.substring(11,16) === 'false') {
-                return 'No permission to download sample'
-            }
-            else{
-                return{
-                        content: res.Bytes,
-                        filename: filename + '.zip',
-                        ec: ec
-                    };
-            }
-        }
-        return res;
-    }
+    try:
+        if demisto.command() == 'test-module':
+            TestgetSession()
+            demisto.results('ok')
 
-    function getReportCommand(){
-        var uriSuffix = jobOrTaskId();
-        var type = args.type ? args.type : 'pdf';
-        var threshold = args.threshold;
+        if demisto.command() == 'atd-login':
+            getSession()
 
-        var filename = args.jobId;
-        if ((!filename) || (filename.length === 0)) {
-            filename = args.taskId;
-        }
-        try{
-            return returnReport(uriSuffix, filename, type, threshold);
-        }
-        catch(error){
-            if (error.message === 'EOF'){
-                throw "The submission doesn't exist in McAfee ATD";
-            }
-        }
-    }
+        elif demisto.command() == 'atd-list-analyzer-profiles':
+            listProfiles()
 
-    function jobOrTaskId(){
-        uriSuffix = '';
-        if ( (!args.jobId && !args.taskId) || (args.jobId && args.taskId) ){
-            throw 'You must specify one (and only one) of the following: jobId, taskId.';
-        }
+        elif demisto.command() == 'atd-list-user':
+            listUsersCommand()
```

Comment:
listUsersCommand => list_users_command (python conventions)

Reply:
done for all commands